src/Aspire.Hosting.Azure.AIFoundry/api/Aspire.Hosting.Azure.AIFoundry.cs

```diff
@@ -37,8 +37,14 @@ public partial class AIFoundryModel
 
         public static partial class AI21Labs
         {
+            [System.Obsolete("Use AI21Jamba15Large instead.")]
+            [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
@sebastienros - this library is in preview mode. Do we need to have Obsolete/EBS.Never properties? Can we just change the names in the major version? How long will we support these old names?

Reply:
I thought we wanted one release with [obsolete] even in a major release, unless you say we don't need if it was "preview" before. We need a flow chart in the doc with these rules ;)

Reply:
https://github.com/dotnet/aspire/pull/12755 should address this.